src/content/plugins-tutorials/native-audio.md

```diff
@@ -53,6 +53,6 @@ NativeAudio.play({
 
 The `preload` method is used to load an audio file into memory, and the `play` method is used to play the loaded audio file.
 
-Other supported methods include `pause`, `resume`, `loop`, `stop`, `unload`, `setVolume`, `getDuration`, and `getCurrentTime`. You can refer to the [official documentation](https://github.com/riderx/native-audio/blob/master/README.md) for more details on these methods.
+Other supported methods include `pause`, `resume`, `loop`, `stop`, `unload`, `setVolume`, `getDuration`, and `getCurrentTime`. You can refer to the [official documentation](https://github.com/riderx/) for more details on these methods.
```

Comment:
same here, why ?

Reply:
This link is really dead.
![Screenshot 2023-09-15 074622](https://github.com/Cap-go/website/assets/81948346/f63451d2-5bef-42c4-86e8-be673cb475d6)

Reply:
good link: https://github.com/Cap-go/native-audio/blob/main/README.md

Reply:
Okk